wix/actyx-installer/Dockerfile

```diff
@@ -1,3 +1,6 @@
+# This image uses WiX 3.8
+# Check the following link for more information:
+# https://hub.docker.com/r/justmoon/wix/dockerfile
 FROM justmoon/wix
```

Comment:
I’d feel better if we depended on the specific hash

Reply:
The image hasn't been updated in something like 8 or 4 years. But I get you